2019/python/generate_2019_error_files.py

```diff
@@ -1,9 +1,27 @@
-#This script will generate a file that will fail a specific syntax, validity, or quality edit (though others may also fail)
+import os
```

Comment:
Good job working out how to make the quality edit test files pass validity edits.

Reply:
Thanks!